src/client/entity-editor/common/new-date-field.js

```diff
@@ -140,10 +140,11 @@ class DateField extends React.Component {
 					groupClassName={groupClassName}
 					label={labelElement}
 				>
-					<InputGroup style={{width: '18em'}}>
+					<InputGroup className="res-dob">
 						<FormControl
 							maxLength={isCommonEraDate ? 4 : 5}
 							placeholder="YYYY"
+							style={{width: '4em'}}
```

Comment:
It was a good move to remove the inline style from the InputGroup, let's make sure we continue with the cleaner solution and put this styling in the css file instead of inlined

Reply:
what about other FormControl components , should i shift their inline styles to style.less as well?

Reply:
If they have a classname that allows them to be targeted with CSS, sure, why not.